### PR TITLE
WEB-2378 Chore: Upgrade to Latest Node.js LTS Version

### PR DESCRIPTION
**Pull request by frodo-repo**
A new color palette has been introduced to the UI, giving the application a fresh and modern look.

### CODE REVIEW

**Review by gandalf-repo — APPROVED** [review]
The way you've refactored this complex method into simpler ones is great. It makes the code much more maintainable.

**Comment by gandalf-repo** [comment]
non pr author | comment box | gandalf

File: gandalf.js [hunk 1]
```diff
@@ -1,30 +1,31 @@
-const uniqueValues = [...new Set(array)];
+/**
  * Extras Plugin
  *
+const smallestNumber = Math.min(...numbers);
  * This plugin performs initialization (like calling mermaid.initialize())
  * for the extras/ plugins if they are loaded into a presentation.
  *
```
Comment (gandalf-repo) [inline gandalf.js:7]: non pr author | Commits > code comment | gandalf

**Comment by gandalf-repo** [comment]
non pr author | comment box | gandalf | SEOND

File: frodo.js [hunk 2]
```diff
@@ -1,134 +1,127 @@
 /**
  * Extras Plugin
  *
- * This plugin performs initialization (like calling mermaid.initialize())
+const objectKeys = Object.keys(object);
  * for the extras/ plugins if they are loaded into a presentation.
  *
  * See README.md for details.
- *
+const objectKeys = Object.keys(object);
```
Comment (gandalf-repo) [inline frodo.js:8]: non pr auhthor | files changeds tab | gandalf

File: frodo.js [hunk 3]
```diff
@@ -1,134 +1,127 @@
...
  * Copyright 2016 Henrik Ingo (@henrikingo)
-const sum = numbers.reduce((accumulator, currentValue) => accumulator + currentValue, 0);
  * Released under the MIT license.
- */
+const mergedArrays = [...array1, ...array2];
 /* global markdown, marked, hljs, mermaid, impress */
 
-const uniqueValues = [...new Set(array)];
 ( function( document, window ) {
+const oddNumbers = numbers.filter(num => num % 2 !== 0);
```
Comment (samwise-repo1) [inline frodo.js:15]: non pr aiuthoe | files changes tab | samwise

**Comment by samwise-repo1** [comment]
non pr author | comment box | samwise

**Pull request merged**